### PR TITLE
etcdctl: alarm command

### PR DESCRIPTION
**Pull request by heyitsanthony**
Adds support for `etcdctl alarm list` and `etcdctl alarm disarm`.

### CODE REVIEW

File: alarm/alarms.go [hunk 1]
```diff
@@ -103,8 +103,16 @@ func (a *AlarmStore) Deactivate(id types.ID, at pb.AlarmType) *pb.AlarmMember {
...
 	a.mu.Lock()
 	defer a.mu.Unlock()
-	for _, m := range a.types[at] {
-		ret = append(ret, m)
+	if at == pb.AlarmType_NONE {
+		for _, t := range a.types {
+			for _, m := range t {
+				ret = append(ret, m)
+			}
+		}
```
Comment (gyuho) [inline alarm/alarms.go]: `return ret` here and remove else-clause?

**Comment by gyuho** [comment]
lgtm. defer to @xiang90

File: etcdctl/ctlv3/command/printer.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ type printer interface {
 	Watch(v3.WatchResponse)
 
 	MemberList(v3.MemberListResponse)
+	Alarm(v3.AlarmResponse)
```
Comment (xiang90) [inline etcdctl/ctlv3/command/printer.go]: empty line between memberList and Alarm?

File: etcdserver/apply.go [hunk 3]
```diff
@@ -414,8 +420,17 @@ func (a *applierV3backend) Alarm(ar *pb.AlarmRequest) (*pb.AlarmResponse, error)
...
+		if !deactivated {
+			break
+		}
+
+		switch m.Alarm {
+		case pb.AlarmType_NOSPACE:
+			plog.Infof("alarm disarmed %+v", ar)
 			a.s.applyV3 = newQuotaApplierV3(a.s, &applierV3backend{a.s})
+		default:
+			plog.Warningf("unimplemented alarm deactivation (%+v)", m)
```
Comment (xiang90) [inline etcdserver/apply.go]: errorf?

**Comment by xiang90** [comment]
lgtm

**Pull request merged**